config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Bundle.java

```diff
@@ -363,4 +366,8 @@ public String getNamespace(final EntityBuilder.BundleType bundleType, boolean is
         return getProjectInfo().getGroupName();
     }
 
+    public static boolean isEnvironmentEntityUniqueNamingDisabled() {
+        return Boolean.getBoolean(DISABLE_ENVIRONMENT_ENTITY_UNIQUE_NAMING);
```

Comment:
I didn't understand how this gives the value of the system property? This is initialized with the property name.

Reply:
I was not known such an API exist with the JDK. I learnt from the existing usage (ServiceAndPolicyLoaderUtil.java).

Reply:
Boolean.getBoolean: returns the value of the system property if it exists. Otherwise, returns false.

Reply:
Awesome!